### ISSUE TITLE
Windows: k: disappeared after starting windows from hibernate

### ISSUE THREAD

**Issue by sirishkumar** [keybase/client#2890]
I cannot see k: after starting windows from hibernate, before hibernating windows I was able to access k:

log id: 7ca15103e46440234e6fbd1c

**Comment by taruti** [comment]
Seems like we are exceeding context deadlines again.

Were you having any network issues or a very slow network in the last 15min before the sleep?

**Comment by sirishkumar** [comment]
@taruti I changed connection between wifi and LAN few hours before the sleep and had a temporary loss of connection.

**Comment by sirishkumar** [comment]
I observed this only one time. cannot reproduce it again.

**Comment by taruti** [comment]
Yes, working on reproducing this locally. Thanks for the report will work on this.

**Comment by sirishkumar** [comment]
I still this issue on windows, any update on this?
